charts/kube-prometheus-stack/hack/sync_grafana_dashboards.py

```diff
@@ -58,8 +58,8 @@ def new_representer(dumper, data):
     'scheduler': ' .Values.kubeScheduler.enabled',
     'node-rsrc-use': ' .Values.nodeExporter.enabled',
     'node-cluster-rsrc-use': ' .Values.nodeExporter.enabled',
-    'nodes': ' .Values.nodeExporter.operatingSystems.linux.enabled',
-    'nodes-darwin': ' .Values.nodeExporter.operatingSystems.darwin.enabled',
+    'nodes': ' (and .Values.nodeExporter.enabled .Values.nodeExporter.operatingSystems.linux.enabled)',
+    'nodes-darwin': ' (and .Values.nodeExporter.enabled .Values.nodeExporter.operatingSystems.darwin.enabled)',
```

Comment:
@QuentinBisson are these intentional for this commit? Looks like they are not related with the issue, from what I understood of the issue.

Reply:
Yes sorry, those are intentional as we removed `.Values.nodeExporter.enabled` as a condition in a previous PR while it does not make sense to have `.Values.nodeExporter.operatingSystems.linux.enabled` generate dashboard when node exporter itself is missing

Reply:
Alright, just checking 😄